src/Coxeter_triangulation/include/gudhi/IO/output_meshes_to_medit.h

```diff
@@ -0,0 +1,151 @@
+                                                                        Mesh_elements& edges, Mesh_elements& triangles,
+                                                                        Mesh_elements& tetrahedra,
+                                                                        Scalar_field_range& triangles_scalar_range,
+                                                                        Scalar_field_range& tetrahedra_scalar_range,
+                                                                        std::size_t index, const Meshes&... meshes) {}
+
+template <std::size_t I = 0, typename... Meshes>
+typename std::enable_if<I != sizeof...(Meshes), void>::type fill_meshes(Vertex_points& vertex_points,
+                                                                        Mesh_elements& edges, Mesh_elements& triangles,
+                                                                        Mesh_elements& tetrahedra,
+                                                                        Scalar_field_range& triangles_scalar_range,
+                                                                        Scalar_field_range& tetrahedra_scalar_range,
+                                                                        std::size_t index, const Meshes&... meshes) {
+  auto mesh = std::get<I>(std::forward_as_tuple(meshes...));
+  for (const auto& v : mesh.vertex_points) vertex_points.push_back(v);
+  for (const auto& e : mesh.edges) {
+    std::vector<std::size_t> edge;
+    for (const auto& v_i : e.first) edge.push_back(v_i + index);
+    edges.emplace_back(std::make_pair(edge, e.second));
+  }
+  for (const auto& t : mesh.triangles) {
+    std::vector<std::size_t> triangle;
+    for (const auto& v_i : t.first) triangle.push_back(v_i + index);
+    triangles.emplace_back(std::make_pair(triangle, t.second));
+  }
+  for (const auto& t : mesh.tetrahedra) {
+    std::vector<std::size_t> tetrahedron;
+    for (const auto& v_i : t.first) tetrahedron.push_back(v_i + index);
+    tetrahedra.emplace_back(std::make_pair(tetrahedron, t.second));
+  }
+  for (const auto& b : mesh.triangles_scalar_range) triangles_scalar_range.push_back(b);
+  for (const auto& b : mesh.tetrahedra_scalar_range) tetrahedra_scalar_range.push_back(b);
+  fill_meshes<I + 1, Meshes...>(vertex_points, edges, triangles, tetrahedra, triangles_scalar_range,
+                                tetrahedra_scalar_range, index + mesh.vertex_points.size(), meshes...);
+}
+
+/** \brief Outputs a text file with specified meshes that can be visualized in Medit.
+ *
+ *  @param[in] amb_d Ambient dimension. Can be 2 or 3.
+ *  @param[in] file_name The name of the output file.
+ *  @param[in] meshes A pack of meshes to be specified separated by commas.
+ */
+template <typename... Meshes>
+void output_meshes_to_medit(std::size_t amb_d, std::string file_name, const Meshes&... meshes) {
```

Comment:
This one has a doc comment, but I don't see it in the generated doc.

Reply:
doc review: fix `build_mesh_from_cell_complex` and `output_meshes_to_medit` doxygen issue on af94bf9d